### ISSUE TITLE
Use yaml format when printing GPU info

### ISSUE THREAD

**Issue by ivaigult** [ARM-software/libGPUInfo#5]
Currently a custom format is used:
```c++
    std::cout << "GPU configuration\n";
    std::cout << "  - Name: " << info.gpu_name << "\n";
    std::cout << "  - Architecture: " << info.architecture_name << "\n";
    std::cout << "  - Model number: 0x" << std::hex << info.gpu_id << std::dec << "\n";
    std::cout << "  - Core count: " << info.num_shader_cores << "\n";
    std::cout << "  - L2 cache count: " << info.num_l2_slices << "\n";
    std::cout << "  - Total L2 cache size: " << info.num_l2_bytes << " bytes\n";
    std::cout << "  - Bus width: " << info.num_bus_bits << " bits\n";
``` 
We can make it look like yaml, if we simply add a colon, and remove a few dashes:
```c++
    std::cout << "GPU configuration:";
    std::cout << "  Name: " << info.gpu_name << "\n";
    std::cout << "  Architecture: " << info.architecture_name << "\n";
    std::cout << "  Model number: 0x" << std::hex << info.gpu_id << std::dec << "\n";
    std::cout << "  Core count: " << info.num_shader_cores << "\n";
    std::cout << "  L2 cache count: " << info.num_l2_slices << "\n";
    std::cout << "  Total L2 cache size: " << info.num_l2_bytes << " bytes\n";
    std::cout << "  Bus width: " << info.num_bus_bits << " bits\n";
```

This would simplify parsing and will ease integration with scripting languages, e.g. Python or JS.

**Comment by solidpixel** [comment]
I view the command line tool as an example of the library usage, producing output designed for human consumption in e.g. log files.

My expectation is that any "real"  use would just integrate the library directly.

**Comment by solidpixel** [comment]
Matching YAML probably meets both needs though, so that's an option.

**Comment by ivaigult** [comment]
I edited the title to mention yaml only. Agree, I don't think we need JSON.

**Comment by solidpixel** [comment]
Fixed in 2e3e19b
